packages/playground/windows/playground-composition/Playground-Composition.cpp

```diff
@@ -175,7 +175,8 @@ struct CompReactPackageProvider
     : winrt::implements<CompReactPackageProvider, winrt::Microsoft::ReactNative::IReactPackageProvider> {
  public: // IReactPackageProvider
   void CreatePackage(winrt::Microsoft::ReactNative::IReactPackageBuilder const &packageBuilder) noexcept {
-    packageBuilder.AddTurboModule(L"DeviceInfo", winrt::Microsoft::ReactNative::MakeModuleProvider<DeviceInfo>());
+    AddAttributedModules(packageBuilder, true);
```

Comment:
Do you need both of these lines at this point.  Shouldn't AddAttributedModules add DeviceInfo?